bitmark-references/bitmark-cli/bitmark-cli.md

```diff
@@ -0,0 +1,729 @@
+# bitmark-cli Command Reference
+
+## Basic Command Structure
+
+~~~
+bitmark-cli [global-options] command [command-options]
+~~~
+
+## Global Options
+
+Options are listed as long form with any single character abbreviation.
+
+`--verbose|-v`
+
+: This allows the CLI to output additional information for each
+command.  Most commands will output the JSON request and response sent
+to the bitmarkd.
+
+`--network|-n  [bitmark|testing|local]`
+
+: Determines which network the command will be sent to from the following possible values:
+* `bitmark` the live network which uses live BTC or LTC to pay for the transactions.
+* `testing` a network for testing newly developed programs, it uses testnet coins to pay for transactions.
+* `local` a special case for running a regression test network on the loopback interface.
+
+`--connection|-c NUMBER`
+
+: Optionally allows for one of the other connections listed in the
```

Comment:
I think it is a little bit hard for people to understand the `connection` concept at the beginning. Do you think it would be helpful if we explain what is a connection is?

Reply:
there needs to be a section on the configuration file and have a reference to the connections array to make this clear